Tasks/AzureRmWebAppDeployment/operations/TaskParameters.ts

```diff
@@ -8,7 +9,7 @@ export class TaskParametersUtility {
             WebAppKind: tl.getInput('WebAppKind', false),
             DeployToSlotOrASEFlag: tl.getBoolInput('DeployToSlotOrASEFlag', false),
             VirtualApplication: tl.getInput('VirtualApplication', false),
-            Package: tl.getPathInput('Package', true),
+            Package: new Package(tl.getPathInput('Package', true)),
```

Comment:
package is not required for container app deployment. We might want to set null in this case.